pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2566,6 +2566,10 @@ self: super: {
     lsp = self.lsp_1_5_0_0;
   };
 
+  autodocodec-yaml = super.autodocodec-yaml.override {
```

Comment:
Can you suggest upstream (and link the issue) that they use stackage, so we don't have to carry this override indefinitely.

Reply:
autodocodec-yaml is already [part of stackage](https://github.com/commercialhaskell/stackage/blob/ab6cd8bc456dd271b3a2c787a888a25e26ebed78/build-constraints.yaml#L4004)

Reply:
Okay, nice.

Reply:
This should be un-done to get rid of the multiple dependencies problem.